garminconnect/__init__.py

```
@@ -716,14 +716,56 @@ def get_personal_record(self) -> Dict[str, Any]:
+                return False
+            if badge["badgeProgressValue"] == badge["badgeTargetValue"]:
+                if (
+                    "badgeLimitCount" not in badge
+                    or badge["badgeLimitCount"] is None
+                ):
+                    return False
+                return badge["badgeEarnedNumber"] < badge["badgeLimitCount"]
+            return True
+
+        earned_in_progress_badges = list(
+            filter(badge_in_progress, earned_badges)
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)